GOVERNANCE.md

```diff
@@ -0,0 +1,97 @@
+# Governance
+
+`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography projects.
```

Comment:
Does 'cryptography projects' mean 'projects wrapping/implementing cryptography' or 'projects using cryptography'? Might want to make this clear.